ab_testing_tool/settings/secure.py.j2

```diff
@@ -8,4 +8,11 @@ SECURE_SETTINGS = {
     'django_db_pass': '{{ secure_settings.django_db_pass }}',
     'canvas_token': '{{ secure_settings.canvas_token }}',
     'django_secret_key': '{{ secure_settings.django_secret_key }}',
+    
+    'email_host': '{{ secure_settings.email_host }}',
```

Comment:
This file is obsolete, but these can be copied over into the new secure.py.example file.